Ex1/ex1.js

```diff
@@ -0,0 +1,11 @@
+//Scrieți o funcție care returnează pătratul unui număr
```

Comment:
```suggestion
// Scrieți o funcție care returnează pătratul unui număr
```
(optional) se lasa un spatiu dupa comentarii